src/analyzer/protocol/ntlm/ntlm-analyzer.pac

```diff
@@ -182,7 +183,10 @@ refine connection NTLM_Conn += {
 
 		if ( ${val}->has_version() )
 			result->Assign(5, build_version_record(${val.version}));
-
+        
+        if ( ${val}->has_response() )
+            result->Assign(6, to_stringval(${val.response.string.data}));
+        
```

Comment:
These new lines are indented with spaces rather than tabs. Could you check your editor settings (e.g. noexpandtab, tabstop=8) and re-align.